tests/run/sys_path_globbed_T5942.srctree

```diff
@@ -0,0 +1,33 @@
+PYTHON setup.py build_ext --inplace
```

Comment:
Please remove the `_T…` part of the file name. We use tags for this.
```suggestion
# tag: gh5942

PYTHON setup.py build_ext --inplace
```

Reply:
Will do. Let's update the hacker documentation (https://github.com/cython/cython/wiki/HackerGuide#bugfeature-tracking-and-project-culture) which says "Either way, before a ticket gets fixed or assigned a milestone, there must be a failing test case in the appropriate `tests/` directory (preferably `tests/run`) that is named `nicely_descriptive_name_here_Txyz.pyx` (where `xyz` is the ticket number)." Is there a way I can do this, or else a ticket I can file somewhere?